app/jobs/remote_server_job.rb

```diff
@@ -96,6 +96,8 @@ def ruby_trunk_discourse(ssh, commit_hash, options)
   end
 
   def rails_releases(ssh, rails_version, options)
+    include_patterns = build_include_patterns(options[:include_patterns])
```

Comment:
Do you want me to remove the build_include_patterns method and just call -e \"INCLUDE_PATTERNS=... directly